### DISCUSSION TITLE
how can magicks be used in the browser

### DISCUSSION THREAD

**Discussion by Christiandrossel** [ImageMagick/ImageMagick#1896] [General] [answered]
Hello,
i would like to use magicks in the browser
(preferably in javascript or typescript).
The topic is new to me, an example project would be very helpful.

**Comment by urban-warrior** [comment, marked as answer]
There are a number of Node.JS projects that utilize ImageMagick.  See https://ciphertrick.com/image-manipulation-using-nodejs-imagemagick/, for example.  Or see https://coderwall.com/p/dvgwtg/node-js-dependency-free-image-processing-with-imagemagick.

**Comment by dlemstra** [comment]
An other alternative would be https://github.com/dlemstra/Magick.WASM. But that project is still in the early stages of development.
